main.go

```diff
@@ -60,13 +60,17 @@ func main() {
 	go fe.Start()
 
 	for {
-		<-w.Events
+		e := <-w.Events
+
+		if e.Op == inotify.Remove || e.Op == inotify.Rename {
+			continue
+		}
 
 		log.Infof("Config has changed: reloading")
 
 		cfg, err := loadConfig(*cfgFilepath)
 		if err != nil {
-			log.Fatalf("Unable to load config: %v", err)
+			log.Errorf("unable to reload config: %v", err)
```

Comment:
should probably continue early here since otherwise you'll attempt to load a broken config?

Reply:
+1 on continue

Reply:
@mraerino not sure what do you mean, can you elaborate?

Reply:
?
```
		cfg, err := loadConfig(*cfgFilepath)
		if err != nil {
			continue
			log.Errorf("unable to reload config: %v", err)
		}
```